### ISSUE TITLE
How to customise whole QMChatCell ?

### ISSUE THREAD

**Issue by MiteshiOS** [QuickBlox/quickblox-ios-sdk#961]
Please let me know how can i customise whole QMChatCell. Also in demo on tap of cell full view of cell is appear but i didn't want like that. I want only message and status in cell. Also provide me suggestion in a way that so i can add some more info if require in future. Also i want to customise message header. Please review the attached screenshot for more information.
![screen shot 2018-03-13 at 3 01 17 pm](https://user-images.githubusercontent.com/17427703/37334035-d173bdd0-26d0-11e8-8457-3d74fb4d2cb2.png)

iOS Version = 11.2.6                                                                                      
Xcode Version = Xcode 9.2

**Comment by Raikerian** [comment]
Hi,

For customization of our components (QMChatViewController in current question), you might need to fork off our main repository of chat controller: https://github.com/QuickBlox/QMChatViewController-ios

In your case please look into QMChatIncomingCell and QMChatOutgoingCell for reference.

**Comment by Pro100Andrey** [comment]
Hi @MiteshiOS, any updates ?

**Comment by MiteshiOS** [comment]
@Pro100Andrey @Raikerian I just tried with possible ways but still not got result that i want. I also took reference from Q-Municate and some how i am able to set msg header(date/time) but main message cell is still complicated to edit for me. Because i want message both status(Read/Delivered) separate from main message and in a cell both are combined. So don't know how to customise same like i want. Any help from you is very appreciate.
